closed/adds/jdk/src/solaris/native/jdk/crypto/jniprovider/NativeCrypto_md.c

```diff
@@ -46,7 +51,8 @@ void * load_crypto_library() {
     "libcrypto.1.1.dylib",
     "libcrypto.1.0.0.dylib",
     "libcrypto.dylib"
-#else
+#elif defined(__linux__)
```

Comment:
I'm not sure it matters, but I'd like to know *why* you added this condition here.

Reply:
Whether or not we change it, I don't think it matters for testing. I'm going to start some builds.

Reply:
This PR aims to add OpenSSL 3.+ for Linux platform rather than AIX/MAC, so I add make an condition for the Linux platform.

Reply:
There's a separate file for Windows, this file needs to support AIX, macOS and Linux. Whether we change this line or not will not affect the result. In my view inertia should win: the line should be left alone.

Reply:
Edited.